.gitignore

```diff
@@ -12,3 +12,6 @@ mmaps/
 
 # automatically generated crates
 /stm32*/
+
+# It's a Mac Thing
+**/.DS_Store 
```

Comment:
```suggestion
```

You can put things like this in a global `.gitignore` (e.g. in your home directory), rather than having to write it in every project: `git config --global core.excludesfile ~/.gitignore` for instance. I think it should be enough to write `.DS_Store` without the leading `**/` too, fwiw.